nixos/modules/installer/tools/tools.nix

```diff
@@ -163,7 +163,7 @@ in
         # List packages installed in system profile. To search, run:
         # \$ nix search wget
         # environment.systemPackages = with pkgs; [
-        #   nano vim # don't forget to add an editor to edit configuration.nix!
+        #   vim # Do not forget to add an editor to edit configuration.nix! The Nano editor is also installed by default.
         #   wget
```

Comment:
I think you included this commit by accident. Please drop it.

Reply:
Weird. I re-based onto master and this diff went away.